ndb/tests/unit/test__api.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest import mock
+
+from google.cloud import _http
+from google.cloud.ndb import _api
+
+
+class TestStub:
+    @staticmethod
+    @mock.patch("google.cloud.ndb._api._helpers")
+    @mock.patch("google.cloud.ndb._api.datastore_pb2_grpc")
+    def test_secure_channel(datastore_pb2_grpc, _helpers):
+        channel = _helpers.make_secure_channel.return_value
+        client = mock.Mock(
```

Comment:
Big 👍

Reply:
Are you really excited about this Mock instance, or did you mean more generally?  ;-)

Reply:
It's just nice to see someone use mock's in a limited fashion (i.e. "correctly")